src/room/InCallView.module.css

```diff
@@ -94,20 +94,6 @@ Please see LICENSE in the repository root for full details.
   justify-self: end;
 }
 
-@media (max-width: 660px) {
-  .footer {
-    grid-template-areas: ". buttons buttons buttons .";
-  }
-
-  .logo {
-    display: none;
-  }
-
-  .layout {
-    display: none !important;
-  }
-}
-
```

Comment:
Removing this will regress https://github.com/element-hq/element-call/pull/2754

Reply:
Looks like a case of `Ctrl-X` rather than `Ctrl-C`